knimin/lib/data_access.py

```diff
@@ -962,7 +970,10 @@ def _decode_zip_lookup(item):
                         'SURVEY_ID'], unknown_external))
             except Exception as e:
                 # Add barcode to error and remove from metadata info
-                errors[barcode] = '%s' % e
+                if isinstance(e, SSLError):
```

Comment:
In fact, it works on my laptop and on Travis, but not on flight. I figure it is due to different lib versions. And to be more precise, it seems to be the "message" object itself which does not work as expected if the string is utf-8. Other Exceptions work as expected, but not SSLError.
I'd be happy if we could defer a cleaner solution and continue working on the pulldown.